src/ResourceManager/Storage/Commands.Management.Storage/StorageAccount/NewAzureStorageAccount.cs

```diff
@@ -176,7 +176,10 @@ public override void ExecuteCmdlet()
             if (this.EnableEncryptionService != null)
             {
                 createParameters.Encryption = ParseEncryption(EnableEncryptionService);
-                createParameters.Encryption.KeySource = "Microsoft.Storage";
+                if (createParameters.Encryption != null)
+                {
+                    createParameters.Encryption.KeySource = "Microsoft.Storage";
+                }
```

Comment:
Can you please update the changelog for the AzureRM.Storage module here: https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Storage/ChangeLog.md

Reply:
@markcowl 
Change log is updated in the PR.